core/src/main/java/nu/yona/server/subscriptions/service/BuddyConnectResponseMessageDto.java

```diff
@@ -137,6 +140,7 @@ MessageActionDto handleAction_Process(UserDto actingUser, BuddyConnectResponseMe
 						connectResponseMessageEntity.getRelatedUserAnonymizedId().get(),
 						connectResponseMessageEntity.getSenderNickname());
 			}
+			UserDto actingUserAfterUpdate = userService.getPrivateUser(actingUser.getId());
```

Comment:
Wouldn't it be fine if you would just inline ``userService.getPrivateUser(actingUser.getId())`` in the place where ``actingUserAfterUpdate`` is used on line 155 below?